packages/change-codemod/Makefile

```diff
@@ -0,0 +1,7 @@
+run-codemod:
```

Comment:
Makefiles make sense internally to normalize setup/run commands across different languages, but I think it might be more idiomatic for a pure JS project to expose these functions as npm scripts e.g. `npm run lodash-to-object-spread`.